pkgs/development/python-modules/taxi/default.nix

```diff
@@ -1,32 +1,34 @@
 { lib
 , buildPythonPackage
-, fetchPypi
+, fetchFromGitHub
 , appdirs
 , requests
 , click
 , setuptools
-, backends ? [ ]
+, pytest
+, freezegun
 }:
 
 buildPythonPackage rec {
   pname = "taxi";
   version = "6.1.1";
 
-  src = fetchPypi {
-    inherit version;
-    pname = "taxi";
-    sha256 = "b2562ed58bd6eae7896f4f8e48dbee9845cd2d452b26dd15c26f839b4864cb02";
+  src = fetchFromGitHub {
+    owner = "sephii";
+    repo = "taxi";
+    rev = version;
+    sha256 = "sha256-iIy3odDX3QzVG80AFp81m8AYKES4JjlDp49GGpuIHLI=";
   };
 
-  # No tests in pypy package
-  doCheck = false;
+  checkInputs = [ pytest freezegun ];
+  checkPhase = "pytest";
 
   propagatedBuildInputs = [
     appdirs
     requests
     click
     setuptools
-  ] ++ backends;
+  ];
 
```

Comment:
```suggestion

  checkInputs = [
    freezegun
    pytestCheckHook
  ];
  
  pythonImportsCheck = [ "taxi" ];

```

Reply:
Should I always have a pythonImportCheck when adding python package even when they have tests?